tests/typescript-css-modules.test.ts

```diff
@@ -76,7 +71,7 @@ describe.sequential('typescript-css-modules', () => {
         args: ['--config=sku-ssr.config.ts'],
       });
       run('node', {
-        args: ['server'],
+        args: ['server.cjs'],
```

Comment:
Now that the `typescript-css-modules` fixture is `type: "module"`, it outputs a `server.cjs` file, but `node server` tries to find `server.js`, so we need an explicit file extension here.